opensciencegrid/ospool-cm/Dockerfile

```diff
@@ -2,17 +2,23 @@
 ARG BASE_OSG_SERIES=3.6
 ARG BASE_YUM_REPO=release
 
-FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el7-$BASE_YUM_REPO
+FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el8-$BASE_YUM_REPO
 
+# base packages
 RUN yum -y upgrade && \
     yum -y install \
-      condor \
       git \
       lsof \
       vim \
     && \
     yum clean all
 
+# we want a newer HTCondor
+RUN yum -y install --enablerepo=osg-upcoming-testing \
+      condor \
+    && \
+    yum clean all
+
```

Comment:
I'm not sure we need to do this. Each tag has its corresponding upcoming repository enabled by default, i.e. `3.6-testing` has `upcoming-testing` enabled.

So if we should be deploying HTCondor 9.5.0, maybe we should just use the `3.6-testing` image instead of `3.6-release`, at least until we officially release it in the OSG Yum repos.

Reply:
Ah, interesting! But the el8 would have to be a change, right?

Reply:
Yeah, we should keep the EL8 change.

One potential reason to keep this particular hunk in would be if we *only* wanted to update condor and its dependencies. I don't think it actually matters in this particular case and lean towards ripping it out because I bet dollars to donuts that we'd forget it was in here :)